barbara/reader.py

```diff
@@ -13,3 +14,18 @@ def read(self) -> OrderedDict:
             return dotenv_values(self.source)
         except TypeError:
             return dotenv_values(self.source.name)
+
+
+class SSMReader:
+    """Reads environment variables from AWS SSM storage into an ordered dictionary"""
+    def __init__(self, prefix, keys):
+        self.prefix = prefix
+        self.keys = keys
+
+    def read(self) -> OrderedDict:
+        environ = OrderedDict()
+        client = boto3.client('ssm')
+        for key in self.keys:
+            result = client.get_parameter(Name=f'{self.prefix}{key}', WithDecryption=True)
```

Comment:
This could get expensive, why not do `getParameters` and retrieve all at once? To catch the failure? Wonder if there's another way to catch that...

Reply:
It's actually super fast, I was suprised!